airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterWorkflowImpl.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling;
+
+import io.airbyte.config.StandardSyncInput;
+import io.airbyte.scheduler.models.IntegrationLauncherConfig;
+import io.airbyte.scheduler.models.JobRunConfig;
+import io.airbyte.workers.temporal.sync.SyncWorkflow;
+import io.temporal.api.enums.v1.ParentClosePolicy;
+import io.temporal.workflow.CancellationScope;
+import io.temporal.workflow.ChildWorkflowOptions;
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConnectionUpdaterWorkflowImpl implements ConnectionUpdaterWorkflow {
+
+  private boolean canStart;
+  private boolean isRunning = false;
+  private boolean isDeleted = false;
+  private final boolean skipScheduling = false;
+  // private Optional<SyncWorkflow> childSync = Optional.empty();
+  CancellationScope syncWorkflowCancellationScope = CancellationScope.current();
```

Comment:
Should this be `private` with the initialization moved to the constructor?

Reply:
I made it `private`, I left the init on line 27 since it is the same for both constructor.